apis/src/websockets/api/game/handler.rs

```diff
@@ -112,6 +113,18 @@ impl GameActionHandler {
                 .handle()
                 .await?
             }
+            GameAction::SetDate(date) => {
+                self.ensure_user_is_player()?;
+                UpdateDateHandler::new(&self.game, self.user_id, &self.pool, date)
+                    .handle()
+                    .await?
+            }
+            GameAction::SetAcceptDate(accept, white_proposer) => {
```

Comment:
this is correct and this is a "bug" but i think this isint a bug deal as per the note, because anyway at any point any of the players can delete a date and there is no check for this (and no notification or interaction with the tourlament organizer for that matter). If we will use this feature to enforce rules, I will need a closer specification of what is needed

Reply:
Also as of now if a schedule expires any player is able to simply schedule again. If we will make schedules more "binding" it would be good to know what is expected

Reply:
What I was trying to say is: The backend should never rely on the user feeding it accurate data. We cannot rely on data the user sends, especially when it gives one user the option to make a decision for another user.
The data we can trust is the `user` we extract from the cookie and we can then trust the `accept/decline` that was delivered via the connection that is authenticated by the cookie. And the color of the user and the proposer needs to be established from the database.